### ISSUE TITLE
Add MPI (Message Passing Interface)

### ISSUE THREAD

**Issue by prachi237** [conda-forge/conda-forge.github.io#1360]
How about adding  ``MPI``  in the glossary section. 
Their is a whole separate section about MPI Variants in Conda-forge. By adding info about MPI in glossary section will help the user understand  it well.

Reference:  https://conda-forge.org/docs/maintainer/knowledge_base.html#message-passing-interface-mpi

**Comment by ForgottenProgramme** [comment]
Agreed!

**Comment by isuruf** [comment]
Not really. MPI is mentioned in only one section and it should be defined there. What's the use of having it being defined in the glossary section?

**Comment by prachi237** [comment]
> Not really. MPI is mentioned in only one section and it should be defined there. What's the use of having it being defined in the glossary section?

Sir,  in glossary section  we can define 'what MPI is' , just like other keywords are defined. As the section talks about MPI in conda-forge specifically not what MPI is 😅

**Comment by isuruf** [comment]
Glossary should be about words that are used everywhere in the docs. It doesn't make sense for MPI to be in the glossary, because MPI is talked about only in one section. Why not defined what MPI is in that section?
I don't think any user looks at the glossary, so there's no use for defining it there.

**Comment by prachi237** [comment]
> Glossary should be about words that are used everywhere in the docs. It doesn't make sense for MPI to be in the glossary, because MPI is talked about only in one section. Why not defined what MPI is in that section?
> I don't think any user looks at the glossary, so there's no use for defining it there.

Yeah I agree, instead of adding it in glossary, defining what MPI is in the section itself would be a great idea.

**Comment by prachi237** [comment]
> Glossary should be about words that are used everywhere in the docs. It doesn't make sense for MPI to be in the glossary, because MPI is talked about only in one section. Why not defined what MPI is in that section?
> I don't think any user looks at the glossary, so there's no use for defining it there.

sir, today I found that MPI is mentioned even in user  docs https://conda-forge.org/docs/user/tipsandtricks.html
